2022/day01/8lall0/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	f, err := os.Open("input.txt")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
```

Comment:
Leaving a comment just to tell you about the available `utils.CheckErr(err)` func. :smile: